frontend/src/views/activity/Activity.vue

```diff
@@ -175,28 +175,28 @@ export default {
       }
       )
     },
-    activityContents () {
-      return this.activity.activityContents()
+    contentNodes () {
+      return this.activity.contentNodes()
     },
     availableContentTypes () {
       return this.category.categoryContentTypes().items.map(cct => ({
         id: cct.id,
         contentType: cct.contentType(),
-        contentTypeNameKey: 'activityContent.' + camelCase(cct.contentType().name) + '.name',
-        contentTypeIconKey: 'activityContent.' + camelCase(cct.contentType().name) + '.icon',
-        contentTypeSort: parseInt(this.$tc('activityContent.' + camelCase(cct.contentType().name) + '.sort')),
-        enabled: true // atct.contentType().allowMultiple || this.countActivityContents(atct.contentType()) === 0
+        contentTypeNameKey: 'contentNode.' + camelCase(cct.contentType().name) + '.name',
+        contentTypeIconKey: 'contentNode.' + camelCase(cct.contentType().name) + '.icon',
+        contentTypeSort: parseInt(this.$tc('contentNode.' + camelCase(cct.contentType().name) + '.sort')),
+        enabled: true // atct.contentType().allowMultiple || this.countContentNodes(atct.contentType()) === 0
       })).sort((a, b) => a.contentTypeSort - b.contentTypeSort)
     }
   },
   methods: {
-    countActivityContents (contentType) {
-      return this.activityContents.items.filter(ac => {
+    countContentNodes (contentType) {
+      return this.contentNodes.items.filter(ac => {
```

Comment:
Funktionsargument `ac` zu `cn` umbenennen

Reply:
good catch

Reply:
Wollen wir das nicht sowieso ausschreiben oder ein "generischen" Namen verwenden (zB item oder in diesem Fall node)

Reply:
Generell stimme ich zu, aber ich denke es braucht immer die Möglichkeit für Ausnahmen. Die JS Arrow Function Syntax ist manchmal etwas verbos wenn man viele funktionale Calls aneinanderreiht.
@manuelmeister eröffnest du ein Issue mit deinen Vorschlägen, damit wir das überall (tritt nicht nur hier auf) umsetzen und dokumentieren können?